.eslintrc.json

```diff
@@ -240,7 +240,7 @@
         "sort-imports": "error",
         "sort-keys": "off",
         "sort-vars": "error",
-        "space-before-blocks": "off",
+        "space-before-blocks": "error",
```

Comment:
Please revert this change.

Reply:
Yes, eslint's rules are blunt objects and are not IMHO fine-grained enough to express the full range of desired formatting. If you look at the code style of this project, you will see that

```js
if (expression) {
  // block
}
else {
  // block
}
```

is the preferred style.